infra/modules/identity-provider/resources/variables.tf

```diff
@@ -1,6 +1,12 @@
-variable "email_identity_arn" {
+variable "domain_name" {
+  description = "The domain name to configure SES"
+  type        = string
+  default     = null
+}
+
```

Comment:
@coilysiren this was an oversight. we had decided to get rid of this variable [in this discussion](https://github.com/navapbc/pfml-starter-kit-app/pull/237#discussion_r1893261498)

Reply:
👀

Reply:
@lorenyu: https://github.com/navapbc/platform-test/pull/158